pipelines/src/main/scala/dagr/pipelines/CreateUnmappedBamFromFastqPipeline.scala

```diff
@@ -53,7 +53,7 @@ class CreateUnmappedBamFromFastqPipeline(
   @Arg(doc="Path to a temporary directory.")                                      val tmp: Path,
   @Arg(flag="o", doc="The output prefix for files that are kept.")                val output: Path,
```

Comment:
I am not satisfied with the dual use of `output`.  Perhaps we use `prefix` and `output`, with the former a `Path` prefix and the latter a directory.